# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/other/PDocAccessible.ipdl
# Commit: b3216f1cfd1f
# Full Hash: b3216f1cfd1f295a707bb089132f86c1e0770c8d
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-02-02 03:35:00
# Regressor Bug: 1688972
# File Overlap Count: 1
# Description:
#   Bug 1688972 - Make FocusedChild work across document/process boundaries. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D103102
# ==============================================================================

diff -r 0a6d78effab1 -r b3216f1cfd1f accessible/ipc/other/PDocAccessible.ipdl
--- a/accessible/ipc/other/PDocAccessible.ipdl	Mon Feb 01 20:09:55 2021 +0100
+++ b/accessible/ipc/other/PDocAccessible.ipdl	Mon Feb 01 17:54:23 2021 +0000
@@ -310,7 +310,7 @@
 
   async TakeFocus(uint64_t aID);
   nested(inside_sync) sync FocusedChild(uint64_t aID)
-    returns(uint64_t aChild, bool aOk);
+    returns(nullable PDocAccessible aResultDoc, uint64_t aResultID);
 
   nested(inside_sync) sync Language(uint64_t aID) returns(nsString aLocale);
   nested(inside_sync) sync DocType(uint64_t aID) returns(nsString aType);